Backend/Program.cs

```diff
@@ -1,25 +1,51 @@
+using Backend.Configurations;
+using Backend.Services;
+
 var builder = WebApplication.CreateBuilder(args);
 
-// Add services to the container.
+// builder.Services.AddDbContext<CoreDbContext>(
+//     opts => opts.UseNpgsql(builder.Configuration["DatabaseConnectionString"])
+// );
+
+// Singletons
+builder.Services.AddSingleton(new MailConfiguration(
+    builder.Configuration["MailHost"]!,

```

Comment:
Maybe put all settings in a common struct thus change settings to sth like "MailServiceSettings: Host"

Reply:
Done. Had to check how to pass nested settings as environment variables.